src/core/mcis/provisioning.go

```diff
@@ -167,6 +167,9 @@ type TbMcisInfo struct {
 	// InstallMonAgent Option for CB-Dragonfly agent installation ([yes/no] default:yes)
 	InstallMonAgent string `json:"installMonAgent" example:"yes" default:"yes" enums:"yes,no"` // yes or no
 
+	// ConfigureCloudAdaptiveNetwork is an option to configure Cloud Adaptive Network (CLADNet) ([yes/no] default:yes)
+	ConfigureCloudAdaptiveNetwork string `json:"configureCloudAdaptiveNetwork" example:"yes" default:"yes" enums:"yes,no"` // yes or no
```

Comment:
```suggestion
	ConfigureCloudAdaptiveNetwork string `json:"configureCloudAdaptiveNetwork" example:"yes" default:"no" enums:"yes,no"` // yes or no
```
아직 특별한 상관은 없지만~ 
까먹기 전에 표기해두었습니다.

Reply:
@seokho-son 

테스트에 필요한 환경은 구성을 완료하였습니다. (정보는 Slack으로 공유 드리겠습니다 ^^)

그런데 테스트를 위해서 간단한 논의가 필요할 것 같습니다.

API를 먼저 추가하고 테스트를 진행하는 방식으로 변경 하면서 환경변수에 Endpoint 정보를 불러보는 부분을 제거했네요 ^^;;;

Reply:
@seokho-son 

(최근 Commit)
API 호출시 Endpoints 가 전달되지 않을 경우 환경변수에서 불러오기를 시도하도록 변경하였습니다 ^^